source/educators/how-tos/share_courses_social_media.rst

```diff
@@ -1,6 +1,6 @@
-#######################################
+####################################
 Sharing Your Courses on Social Media
-#######################################
+####################################
 
```

Comment:
Please remove the `.. only:: Open_edX` line at L10, and remove all content under `.. only: Partners`

Reply:
Please remove the .. only:: Open_edX line at L10